schedule/functional/extra_tests_textmode.yaml

```diff
@@ -66,6 +66,7 @@ conditional_schedule:
                 - '{{steamcmd}}'
                 - console/libqca2
                 - console/python_scientific
+                - console/vmstat
```

Comment:
C&P
Remove this line, it's in the mau-extratests-phub because there is phub added at beginning.
Also schedule it only there, remove it from schedule/functional/extra_tests_textmode.yaml

Reply:
This is scheduled for Tumbleweed and Leap.

Reply:
But not only there ... https://openqa.suse.de/tests/5975595

Reply:
My bad it's ok, it's opnsuse part ...